solo-io/gloo
# Delete internal API docs after generation

### Description
Delete external API docs after generation

### Docs changes
Delete the generated internal API docs after generation

~~May require a manual update of existing files in firebase~~ Confirmed with docs that it does not.

## Testing steps
Run `make go-generate-apis fmt`

validate that the removed files are no longer present on the file-system:

```
ls -l docs/content/reference/api/github.com/solo-io/gloo/projects/gloo/api/external/
ls: docs/content/reference/api/github.com/solo-io/gloo/projects/gloo/api/external/: No such file or directory
```

Notes for reviewers
There isn't a good way to validate this e2e/in CI without running against production, so we will need to run this through ci after review merge, ideally with https://github.com/k8sgateway/k8sgateway/pull/10334 which needs the same validation.

## Checklist:
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 I have made corresponding changes to the documentation
 I have added tests that prove my fix is effective or that my feature works
